src/vellum/workflows/nodes/core/retry_node/node.py

```diff
@@ -72,20 +75,23 @@ def run(self) -> BaseNode.Outputs:
                     terminal_event.error.message,
                     code=terminal_event.error.code,
                 )
+                time.sleep(self.interval)
```

Comment:
[blocking] let's make the parameter optional that defaults to None, then only run time.sleep if defined:
![Screenshot 2025-01-30 at 11 19 23 PM](https://github.com/user-attachments/assets/3269d967-87eb-455f-9844-d92eb7ea90b2)